flow360/component/validator.py

```diff
@@ -26,6 +27,8 @@ def _get_url(self):
             return "validator/surfacemesh/validate"
         if self is Validator.CASE:
             return "validator/case/validate"
+        if self is Validator.GEOMETRY:
+            return "validator/geometry/validate"
```

Comment:
do we have geometry validator? I think this should be removed

Reply:
Removed